src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,15 @@
+$calendar-gap: 1px;
+$border-field: 1px;
+$font-size: 30px;
+$field-width: 100px;
+$space: calc(#{$field-width} + #{$calendar-gap});
+$calendar-width: calc(#{$space} * 7);
```

Comment:
You need to change those calculations a bit. When having 7 columns in a calendar, you should have 6 gaps